features/load-rest.feature

```diff
@@ -2,7 +2,8 @@ Feature: Manage WordPress through endpoints locally
 
   Background:
     Given a WP install
-    And I run `wp plugin install rest-api --activate`
+	When I run `wp core version`
+	Then STDOUT should be a version string >= 4.7
```

Comment:
behat doesn't require spaces. It only requires that related indentation be the same number of chars. If you are suggest this is a coding-style rule, then the coding style rules need to be updated to indicate that tabs are disallowed in xxx, xxx, xxx set of files. The php rules at https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#indentation declare tabs should be used. Since this is the primary language used for wp-cli developers, it seems odd to me to change the indentation rule which then creates an additional burden on devs and reviewers both. Consistency helps efficiency and decreases error rates.